go.sum

```diff
@@ -2,88 +2,87 @@ github.com/Azure/azure-pipeline-go v0.2.1 h1:OLBdZJ3yvOn2MezlWvbrBMTEUQC72zAftRZ
 github.com/Azure/azure-pipeline-go v0.2.1/go.mod h1:UGSo8XybXnIGZ3epmeBw7Jdz+HiUVpqIlpz/HKHylF4=
```

Comment:
💡 I think there will be required changes in like: `builder/Dockerfile` : https://github.com/Azure/aks-periscope/blob/master/builder/Dockerfile 

I think either if we are keeping `mode=vendor` then we might have to add `go mod vendor` or  change line 5 : https://github.com/Azure/aks-periscope/blob/master/builder/Dockerfile#L5 to `-mod=mod`

Reply:
I've added a new commit about the dockerfile, please let me know

Reply:
All good, I will try it out as well. 

FYI: I did added PR for your fork for the changes I recommended to make it easy 🙏 https://github.com/arnaud-tincelin/aks-periscope/pull/2